pow.py

```diff
@@ -0,0 +1,38 @@
+import hashlib
+#create a block class
+class Block:
+    #constructor the block class
+    def __init__(self,data,prevHash):
+        self.data=data
+        self.prevHash=prevHash
+        self.hash=self.calcHash()
+    #calculatng the hash
+    def calcHash(self):
+        hash1 = hashlib.sha256()
+        hash1.update(self.data.encodde('utf-8'))
+        return hash1.hexdigest()
+    #creating the blockchin   
+    class BlockChain:
+        #constructor for the blockchain
+        def __init__(self):
+            self.chain =[self.CreateGensisBlock()]
+            def CreateGesisBlock(self):
```

Comment:
define `create_genesis_block()` outside `__init__`
also `create_genesis_block()` will only be executed once in a lifetime so maybe don't create a function for it